xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/variablesInit.vm

```diff
@@ -98,8 +98,11 @@
 
   @panel-bg:                            $theme.panelBackgroundColor;
   @panel-default-text:                  $theme.panelTextColor;
-  @xwiki-panel-header-bg:               $theme.panelHeaderBackgroundColor;
-  @xwiki-panel-header-text:             $theme.panelHeaderTextColor;
+  @panel-header-bg:                     $theme.panelHeaderBackgroundColor;
+  @panel-header-text:                   $theme.panelHeaderTextColor;
+  ## Kept for backward compatibility
+  @panel-header-bg:                     $theme.panelHeaderBackgroundColor;
+  @panel-header-text:                   $theme.panelHeaderTextColor;
```

Comment:
Wrong less variable name, this should be `@xwiki-panel-header-bg`/`-text`.

Reply:
Sorry, I did not check my changes properly before commiting. Fixed in https://github.com/xwiki/xwiki-platform/pull/2497/commits/dc4c9d593fead0265fddad2e214ce066f44c344b